Library/Homebrew/cmd/search.rb

```diff
@@ -141,13 +141,20 @@ def search_tap(user, repo, rx)
     []
   end
 
-  def search_formulae(rx)
+  # @param [Regexp] rx
+  # @param [Hash] options
+  # @option options [Boolean] :installed If true, include installed formulae
```

Comment:
Yes, to have an option to suppress the `(installed)` suffix would be useful to make the output of some commands more useful when used programmatically. I don't necessarily think this has to be part of this PR though. Once the option handling is in place, it's easy to add an additional option (and the code that uses it) in a separate PR.